gluster-exporter/main.go

```diff
@@ -113,6 +120,7 @@ func main() {
 	http.Handle(metricsPath, promhttp.Handler())
 	if err := http.ListenAndServe(fmt.Sprintf(":%d", port), nil); err != nil {
 		_, _ = fmt.Fprintf(os.Stderr, "Failed to run exporter\nError: %s", err)
+		log.WithError(err).Fatal("Failed to run exporter")
```

Comment:
`os.Exit` can be removed in next line if we use `.Fatal` here

Reply:
Ack